provision-contest/ansible/roles/mysql_replication/templates/setup-replication.sh.j2

```diff
@@ -117,7 +117,7 @@ if check_replication ; then
 fi
 
 echo "Stopping PHP locally..."
-service php7.4-fpm stop
+service {{ php_version.stdout }}-fpm stop
```

Comment:
Shouldn't this be `php{{ php_version.stdout }}-fpm stop` lie the change above? Also see below.

Reply:
Yes good catch... This went wrong in copy/paste